### PR TITLE
(fix): Adding accountId parameter on user

### PR DESCRIPTION
**Pull request by LKhoe**
Hello, I found a problem in user function and fixed it.
See you.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -40,6 +40,11 @@ Jira Python Library
         :alt: Requirements Status
 
 
+Note
+----------
+It's just and fix for the original lib
```
Comment (russoz) [inline README.rst:45]: This text does not add much to the end user, IMO.

File: jira/client.py [hunk 2]
```diff
@@ -2630,7 +2642,7 @@ def user(self, id, expand=None):
         :rtype: User
         """
         user = User(self._options, self._session)
-        params = {}
+        params = {'accountId': id}
```
Comment (russoz) [inline jira/client.py:2645]: The "accountId" change happened a couple of years ago, but I am not sure (not that I looked for  it) what is the backward compatibility policy of this package.

So, it should be interesting not to force that upon all users.

File: setup.cfg [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = jira
+name = jira_333
```
Comment (russoz) [inline setup.cfg:2]: why?

File: setup.py [hunk 4]
```diff
@@ -3,4 +3,4 @@
 
 
 if __name__ == "__main__":
-    setuptools.setup(use_scm_version=True)
+    setuptools.setup(version='3.4.0')
```
Comment (russoz) [inline setup.py:6]: Why are we pinning down a specific version here?

File: jira/client.py [hunk 5]
```diff
@@ -763,6 +763,18 @@ def find(self, resource_format, ids=None):
         resource.find(ids)
         return resource
 
+    def get_changelogs_from_issue(self, issueID_or_key):
```
Comment (adehad) [inline jira/client.py:766]: these functions do not follow the style of the rest of the class.

Here we could use `self._get_json()` with the default `base` argument, the next function can use the argument `base=AGILE_BASE_URL`

**Review by adehad — CHANGES_REQUESTED** [review]

**Pull request closed without merging**